src/pandablocks/asyncio.py

```diff
@@ -50,6 +51,22 @@ async def close(self):
         await writer.wait_closed()
 
 
+class FlushMode(Enum):
+    """
+    The mode which `AsyncioClient.data()` uses when flushing data frames.
+
+    ``IMMEDIATE``: Flush all data frames immediately.
+
+    ``PERIODIC``: Flush data frames periodically.
+
+    ``MANUAL``: Flush data frames when the user sets an `asyncio.Event`.
+    """
+
+    IMMEDIATE = 0
+    PERIODIC = 1
+    MANUAL = 2
```

Comment:
Did you try this as a doc comment?

```suggestion
    """
    #: Flush all data frames immediately
    IMMEDIATE = 0
    #: Flush data frames periodically
    PERIODIC = 1
    #: Flush data frames when the user sets an `asyncio.Event`
    MANUAL = 2
```

Reply:
Thank you, this works.